src/main.js

```diff
@@ -83,7 +84,14 @@ class PercyClient {
       agent: this._httpAgent,
     };
 
-    return this._httpClient(uri, requestOptions);
+    return retry(this._httpClient, {
+      context: this,
+      args: [uri, requestOptions],
+      interval: 50,
```

Comment:
Would maybe be nice to see the explicit number of retries that are happening, so we can tune that over time.